File: 02/part1.py
input = open('input.txt').readlines()
input = map(lambda s: s.replace('\n', ''), input)

depth = 0
horizontal = 0

for string in input:
    replaced = string.replace('down ', '') if 'down' in string else string.replace('up ', '-')
    if replaced != string:
        depth += int(replaced)
    else:
        horizontal += int(string.replace('forward ',''))

print(depth*horizontal)

File: 01/part1.pl
open my $file, '<', 'input.txt';

my @input;

while (<$file>) {
    chomp;
    push @input, int $_;
}
close $file;

my $increases = 0;

for (my $i = 1; $i < scalar @input; $i++) {
    $increases += @input[$i] > @input[$i-1];
}

print $increases;
